crates/cxx-qt-gen/test_outputs/qenum.rs

```diff
@@ -18,6 +18,7 @@ mod ffi {
         include!("directory/file_ident.cxxqt.h");
     }
     #[repr(i32)]
+    #[namespace = "cxx_qt::my_object"]
```

Comment:
so before these were getting the namespace from the bridge level one, guess we are now explicit.

But i note looking at some of the other outputs not everything has an explicit namespace, looks like signals are missing them? (if we are going down the everything we generate has an explicit namespace)